calculadora-idade.js

```diff
@@ -0,0 +1,37 @@
+function calculaIdade(dia,mes,ano){
+    const anoCorrente = 2023;
+    const mesCorrente = 11;
+    const diaCorrente = 27;
+
+    const subAno = anoCorrente - ano
+    const subMes = mesCorrente - mes
+    const subDia = diaCorrente - dia
+
+
+     
+     if(subAno <= 0)
+         return ("Insira um ano válido");
+
+    if (subMes > 0) 
+    return subAno     
+     
+    if(subMes < 0) {
+         return subAno - 1;
+    }
+    else {
+             if(subDia  >0 ){
```

Comment:
Aqui vc pode usar `if(subDia >= 0)` pois tanto se for maior que zero, ou igual, a pessoa já fez o aniversário. (Lembrando que se for igual significa que estamos no mesmo dia)